src/app/onboard/name/page.tsx

```diff
@@ -0,0 +1,56 @@
+"use client";
+
+import { useState } from "react";
+
+import UabcHeaderText from "@/components/AuthLogin/UabcHeaderText";
+import { Button } from "@/components/Button";
+import { TextInput } from "@/components/TextInput";
+import { useAccountStore } from "@/stores/accountDetails";
+
+export default function NamePage() {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+
+  const updateFirstName = useAccountStore((state) => state.setFirstName);
+  const updateLastName = useAccountStore((state) => state.setLastName);
```

Comment:
bit unnecessary to be having firstName and lastName useState hooks if we already have a hook for that, can you delete the useState and use the Store instead?